### DISCUSSION TITLE
Make virtual opcodes have values > 256

### DISCUSSION THREAD

**Discussion by iritkatriel** [faster-cpython/ideas#337] [General]
I think it would be good to change the type of struct instr's i_opcode from unsigned char to int, and make the virtual opcodes have values > 256.  This way we are not wasting opcodes on the virtual ones, so we can add as many virtual ones as we want to improve clarity. Also, it will be easy to detect bugs where a virtual opcode was not converted before the bytecode is assembled.

**Comment by brandtbucher** [comment]
Agreed!
